helios.js

```diff
@@ -1,2 +1,25 @@
 // don't need @deno-types because everything is in JSDocs
-export { Address, Assets, ConstrData, Datum, TxId, IntData, MintingPolicyHash, NetworkParams, Program, PubKeyHash, Value, UTxO, Tx, TxOutput, TxWitnesses, Signature, highlight, bytesToHex, hexToBytes, UplcData, UplcProgram } from "https://www.hyperion-bt.org/Helios/helios.js";
+export {
+    Address,
+    Assets,
+    ConstrData,
+    Datum,
+    IntData,
+    MintingPolicyHash,
+    NetworkParams,
+    Program,
+    PubKeyHash,
+    Signature,
+    Tx,
+    TxId,
+    TxOutput,
+    TxWitnesses,
+    UTxO,
+    UplcData,
+    UplcProgram,
+    Value,
+    bytesToHex,
+    bytesToText,
+    hexToBytes,
+    highlight,
+} from "https://www.hyperion-bt.org/Helios/helios.js";
```

Comment:
I think it makes this file more easily readable and the diff less impacted by future changes.